examples/tests/sdk_tests/testing/assert.main.w

```diff
@@ -0,0 +1,62 @@
+bring util;
+bring expect;
+
+// Num tests
+
+let a: num? = nil;
+let b: num? = 1;
+
+expect.equal(1, 1);
+expect.equal(b, 1);
+expect.isNil(a);
```

Comment:
what are your thoughts about `isX` vs `x`, e.g. `expect.nil` or `expect.isNil`?

Reply:
```js
expect.nil(bang);
```

I would go with `nil()`.

Reply:
What about the reverse one? `expect.notNil(opt)` or `expect.defined(opt)`?

Reply:
added `notNil` as well